client/app/scripts/utils/web-api-utils.js

```diff
@@ -167,11 +166,8 @@ function getNodesForTopologies(getState, dispatch, topologyIds, topologyOptions
     .filter((_, topologyId) => topologyIds.contains(topologyId))
     .reduce((sequence, topologyUrl, topologyId) => sequence.then(() => {
       const optionsQuery = buildOptionsQuery(topologyOptions.get(topologyId));
-      // Trim the leading slash from the url before requesting.
-      // This ensures that scope will request from the correct route if embedded in an iframe.
-      return fetch(`${trimStart(topologyUrl, '/')}?${optionsQuery}`);
+      return doRequest({ url: `${getApiPath()}${topologyUrl}?${optionsQuery}` });
```

Comment:
No, `trimStart` alone wasn't the problem as it didn't help when I simply removed it. Maybe it's about the additional CSRF thingy that `doRequest` does?